<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigation</class>
 <widget class="QWidget" name="Navigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>70</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigation</string>
  </property>
  <widget class="Marble::ArrowDiscWidget" name="arrowDisc" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="Marble::NavigationButton" name="homeButton" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>70</y>
     <width>34</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="icon" stdset="0">
    <iconset resource="navigation.qrc">
     <normaloff>:/marble/navigation/navigational_homebutton.png</normaloff>
     <activeoff>:/marble/navigation/navigational_homebutton_hover.png</activeoff>
     <selectedoff>:/marble/navigation/navigational_homebutton_press.png</selectedoff>:/marble/navigation/navigational_homebutton.png</iconset>
   </property>
   <property name="iconSize" stdset="0">
    <size>
     <width>34</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="Marble::NavigationButton" name="zoomInButton" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>109</y>
     <width>34</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="icon" stdset="0">
    <iconset resource="navigation.qrc">
     <normaloff>:/marble/navigation/navigational_zoom_plus.png</normaloff>
     <disabledoff>:/marble/navigation/navigational_zoom_press_plus.png</disabledoff>
     <activeoff>:/marble/navigation/navigational_zoom_hover_plus.png</activeoff>
     <selectedoff>:/marble/navigation/navigational_zoom_press_plus.png</selectedoff>:/marble/navigation/navigational_zoom_plus.png</iconset>
   </property>
   <property name="iconSize" stdset="0">
    <size>
     <width>34</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="Marble::NavigationButton" name="zoomOutButton" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>305</y>
     <width>34</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="icon" stdset="0">
    <iconset resource="navigation.qrc">
     <normaloff>:/marble/navigation/navigational_zoom_minus.png</normaloff>
     <disabledoff>:/marble/navigation/navigational_zoom_press_minus.png</disabledoff>
     <activeoff>:/marble/navigation/navigational_zoom_hover_minus.png</activeoff>
     <selectedoff>:/marble/navigation/navigational_zoom_press_minus.png</selectedoff>:/marble/navigation/navigational_zoom_minus.png</iconset>
   </property>
   <property name="iconSize" stdset="0">
    <size>
     <width>34</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="Marble::NavigationSlider" name="zoomSlider" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>145</y>
     <width>34</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Marble::ArrowDiscWidget</class>
   <extends>QWidget</extends>
   <header>ArrowDiscWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Marble::NavigationButton</class>
   <extends>QWidget</extends>
   <header>NavigationButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Marble::NavigationSlider</class>
   <extends>QWidget</extends>
   <header>NavigationSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="navigation.qrc"/>
 </resources>
 <connections/>
</ui>
